exercises/strings/rotN/rotN.js

```diff
@@ -15,15 +15,37 @@
  * @returns {string} A lowercase copy of the input string
  */
 
-function rotN(string) {
+function rotN(string, n) {
   // This is your job. :)
```

Comment:
You can remove these comments, BTW. They add noise to the final code and are just there to direct students' attention when first starting out.